apiBase/errors/ValidationError.js

```diff
@@ -1,20 +1,19 @@
 import FakeError from './FakeError';
 
-const msgText = (apiName, errors, model) =>
-  `${ apiName } had errors in ${ errors.join(',') } with properties ${ JSON.stringify(model.toJSON()) }`;
+const msgText = (api, errors) => `${api} had errors in ${errors.join(',')}`;
 
 export default class ValidationError extends FakeError {
-  constructor (apiName, model, errors) {
+  constructor (apiName, errors, status) {
```

Comment:
🔕 We should tweak some combination of validators and the validation in `APIRequestUtils` to give more helpful errors here.